src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -481,6 +481,18 @@ export function pickMutationEnrichmentProfiles(profiles: MolecularProfile[]) {
     );
 }
 
+export function pickStructuralVariantEnrichmentProfiles(
+    profiles: MolecularProfile[]
+) {
+    return _.filter(
+        profiles,
+        (profile: MolecularProfile) =>
+            profile.molecularAlterationType ===
```

Comment:
@kalletlak , problem was here.  you probably know this but you can't use OR operator like this.    has to be   
```
x == y || x === z
```

Reply:
fixed in subsequent commit

Reply:
thanks for fixing